src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -228,28 +234,42 @@ export default function SnapshotListModal() {
               <Tbody>
                 {snapshotsList.map(
                   (
-                    { uuid: snap_uuid, created_at, content_counts, added_counts, removed_counts }: SnapshotItem,
+                    {
+                      uuid: snap_uuid,
+                      created_at,
+                      content_counts,
+                      added_counts,
+                      removed_counts,
+                    }: SnapshotItem,
                     index: number,
                   ) => (
                     <Tr key={created_at + index}>
                       <Td>{new Date(created_at).toUTCString()}</Td>
                       <Td>
-                        <ChangedArrows
-                          addedCount={
-                            (added_counts?.['rpm.advisory'] || 0) +
-                            (added_counts?.['rpm.package'] || 0)
-                          }
-                          removedCount={
-                            (removed_counts?.['rpm.advisory'] || 0) +
-                            (removed_counts?.['rpm.package'] || 0)
-                          }
-                        />
+                        {snap_uuid && (
```

Comment:
You don't need to conditionally show this section. 
And if you do, do not use  `{snap_uuid &&`
[Read this](https://medium.com/@davidkelley87/avoiding-in-reacts-conditional-rendering-dec9eb61ee65#:~:text=Using%20%E2%80%9C%26%26%E2%80%9D%20to%20conditionally%20render%20components%20can%20lead%20to%20unexpected,evaluate%20to%20the%20expected%20value.)